### PR TITLE
Return a conflict response if trying to override an existing template.

### PR DESCRIPTION
**Pull request by gquemener**

### CODE REVIEW

File: Controller/AssistantController.php [hunk 1]
```diff
@@ -26,15 +26,21 @@ public function missingViewAction($viewName, $viewParams)
...
+    public function createViewAction()
     {
-        $viewName = $request->request->get('viewName');
-        $viewBody = $request->request->get('viewBody');
-        $viewPath = $this->get('knp_rad.view.path_deducer')->deducePath($viewName);
-
-        // in case directory does not exist
-        $this->get('filesystem')->mkdir(dirname($viewPath));
-
+        $request    = $this->getRequest();
```
Comment (docteurklein) [inline Controller/AssistantController.php:31]: why not using injected $request  above ? It avoids hard vetical coupling with base controller.

Comment (docteurklein) [inline Controller/AssistantController.php:31]: bah, a detail :) thanks!

File: Controller/AssistantController.php [hunk 2]
```diff
@@ -26,15 +26,21 @@ public function missingViewAction($viewName, $viewParams)
         );
     }
 
-    public function createViewAction(Request $request)
+    public function createViewAction()
```
Comment (lsmith77) [inline Controller/AssistantController.php:29]: i think its cleaner to inject the request explicitly

Comment (gquemener) [inline Controller/AssistantController.php:29]: Ah yes? I thought it was a bad practice... Don't remember why

Comment (lsmith77) [inline Controller/AssistantController.php:29]: i think injecting the container is a bad practice so it should be avoided as much as possible to write code that depends on this. i realize that the base controller does provide the container, but that doesnt mean one should use it when its not necessary.

Comment (docteurklein) [inline Controller/AssistantController.php:29]: Totally agree with @lsmith77

**Pull request merged**